conjure-java-retrofit2-client/src/test/java/com/palantir/conjure/java/client/retrofit2/Retrofit2ClientApiTest.java

```diff
@@ -320,7 +320,7 @@ private void connectionFailureWithFuture(Supplier<Future<String>> futureSupplier
                 AGENT,
                 new HostMetricsRegistry(),
                 ClientConfiguration.builder()
-                        .from(createTestConfig("https://invalid.service.dev"))
+                        .from(createTestConfig("https://service.invalid"))
```

Comment:
The `.dev` TLD was causing test failures in CircleCI. Something must have changes and now this domain correctly resolves and whatever server is there returns a 404.

Switched to `.invalid` which is a reserved TLD for invalid domains.
https://en.wikipedia.org/wiki/.invalid